src/ip/program_loader/README.md

```diff
@@ -0,0 +1 @@
+# PROGRAM LOADER
```

Comment:
Move the program loader files to verif/src

Reply:
Done.